cloud-agent/service/server/src/main/scala/org/hyperledger/identus/issue/controller/IssueControllerImpl.scala

```diff
@@ -69,7 +65,7 @@ class IssueControllerImpl(
                 pairwiseHolderDID = offerContext.pairwiseHolderDID,
                 kidIssuer = request.issuingKid,
                 thid = DidCommID(),
-                maybeSchemaId = request.schemaId,
+                maybeSchemaIds = Applicative[Option].map2(request.schemaIds, request.schemaId.map(List(_)))(_ ++ _),
```

Comment:
@CryptoKnightIOG, two fields might contain schemaId: `schemaId` and `schemaIds`.
Remember, I recommended splitting VCDM 1.1 and 2.0 at the domain level, but I see that the API also should be segregated.
Otherwise, the result API is going to be overloaded and slightly weird.

Reply:
If you are still going to go this way, use the same REST API for both models. I recommend deleting `schemaIds` and allowing `schemaId` to be either String or List[String].

Reply:
I fixed it this PR https://github.com/hyperledger/identus-cloud-agent/pull/1368